src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -135,10 +135,10 @@ public AudioResponse upload(@RequestParam("audio") MultipartFile file,
         criterion.setSrcPath("C:\\_pg\\tmp\\uploads\\" + fileName);
```

Comment:
The variable 'criterion' must be changed to 'criteria' in all classes and methods, as we left in group meeting.

Reply:
this class and methods are changed in the next branch and it is irrelevant to make changes now